search-ui/src/SearchComponent.js

```
@@ -468,7 +549,9 @@ function SearchComponent() {
                 }}
               >
                 <span
-                  dangerouslySetInnerHTML={{ __html: result._formatted.parent_title }}
+                  dangerouslySetInnerHTML={{
+                    __html: result._formatted.parent_title,
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)